tests/models/test_utils.py

```diff
@@ -0,0 +1,82 @@
+import re
+import pytest
+from pydantic import BaseModel
+from optimade.models.utils import OptimadeField, StrictField, SupportLevel
+from typing import List
+
+
+def test_strict_field():
+    """Test :class:`StrictField` creation for failure on bad keys, and
+    warnings with no description.
+
+    """
+    detail = re.escape(
+        "Not creating StrictField ((Ellipsis,), {'random_key': 'disallowed'}) with forbidden keywords ['random_key']."
+    )
```

Comment:
Try if this works.
```suggestion
    detail = (
        r"Not creating StrictField \(\(Ellipsis,\), \{'random_key': 'disallowed'\}\) with forbidden keywords \['random_key'\]\."
    )
```

Reply:
That's what I tried first (well, second) and should be basically what `re.escape(...)` is doing anyway, so I'm a bit stumped for now...